README.md

```diff
@@ -23,13 +23,13 @@ Basic Concept
 
 - Evaluating [SNARE](https://github.com/mushorg/snare) events.
 - Serve dorks.
-- Adopt and change the responses.
+- Emulate vulnerabilities and provide responses.
 
 
 Getting Started
 ---------------
 
-- You need Python3.6 and above for installing tanner.
+- You need Python3.7 and above for installing tanner.
```

Comment:
we don't want to support python 3.6 anymore?

Reply:
3.6 is no longer supported: https://en.wikipedia.org/wiki/History_of_Python#Table_of_versions but your call if you want to keep it at 3.6

Reply:
 😱 I didn't know that 😄